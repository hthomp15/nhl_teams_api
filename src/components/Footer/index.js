import React from "react";

function Footer() {
    return (
        <footer className="mt-6">
            <p className="text-center"> Made with <span role="img" aria-label="heart">❤️</span> by Haile Thompson, May 2022</p>
        </footer>
    );
}

export default Footer;