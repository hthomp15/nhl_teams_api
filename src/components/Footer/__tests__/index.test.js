import React from "react";
import { render, cleanup } from "@testing-library/react";
import '@testing-library/jest-dom/extend-expect';

import Footer from "..";

afterEach(cleanup);

describe("Footer Component", () => {
    it("renders", () => {
        render(<Footer />);
    });
    it("matches snapshot DOM node structure", () => {
        const { asFragment } = render(<Footer />);
        expect(asFragment()).toMatchSnapshot();
    })
    it("inserts Emoji into <p>", () => {
        const { getByLabelText } = render(<Footer />);
        expect(getByLabelText("heart")).toHaveTextContent('❤️')
        })
});