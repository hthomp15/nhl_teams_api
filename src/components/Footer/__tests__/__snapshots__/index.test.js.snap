// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <footer
    class="mt-6"
  >
    <p
      class="text-center"
    >
       Made with 
      <span
        aria-label="heart"
        role="img"
      >
        ❤️
      </span>
       by Haile Thompson, May 2022
    </p>
  </footer>
</DocumentFragment>
`;
